### ISSUE TITLE
Manual: Error: unresolved olink: targetdoc/targetptr = 'manual/module-taskserver'.

### ISSUE THREAD

**Issue by NotaseCretagen** [NixOS/nixpkgs#15108]
## Issue description

```
$ time nix-build /etc/nixos/nixpkgs/nixos/release.nix -A manual.x86_64-linux
these derivations will be built:
  /nix/store/j1m5l6jaf3893l5azmd60bnd08i7z6mj-nixos-manual.drv
building path(s) ‘/nix/store/aswvb9imh7928inyp459l2qrppn8w2lq-nixos-manual’
manual.xml validates
warning: failed to load external entity "olinkdb.xml"
Olink error: could not open target database 'olinkdb.xml'.
Error: unresolved olink: targetdoc/targetptr = 'manual/module-taskserver'.
Writing options.html for appendix(ch-options)
Writing release-notes.html for appendix(ch-release-notes)
Writing index.html for book(book-nixos-manual)
/nix/store/aswvb9imh7928inyp459l2qrppn8w2lq-nixos-manual

real    0m26.107s
user    0m1.857s
sys 0m0.373s
```

Problem seems to be here:
```grep
nixos/modules/services/misc/taskserver/default.nix:197:          <olink targetdoc="manual" targetptr="module-taskserver"/>.
nixos/modules/services/misc/taskserver/doc.xml:4:    xml:id="module-taskserver">
```

## Technical details

* System: 16.09.git.32b7b00M (Flounder)
* Nix version: nix-env (Nix) 1.11.2
* Nixpkgs version: "16.09.git.cf34a7b"

**Comment by NotaseCretagen** [comment]
Btw, [that](https://github.com/NixOS/nixpkgs/blob/de0847c7310c88a68ee6a19012a296db9f66b986/nixos/modules/services/misc/taskserver/default.nix#L197) is the only place that has an `<olink`

**Comment by joachifm** [comment]
Possibly related: https://github.com/NixOS/nixpkgs/pull/14700

**Comment by aszlig** [comment]
Fixed by d2c64031a0dcc48f1755fa1f356e61b0da2bd786.
